php-cs-test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer
+
+RUN composer global require prooph/php-cs-fixer-config
+
+COPY rules.php /rules.php
+
+VOLUME /sut
+WORKDIR /sut
+
+ENTRYPOINT ["/composer/vendor/bin/php-cs-fixer", "fix", "--dry-run", "--diff", "--config=/rules.php"]
```

Comment:
I can do, I was more thinking this image is the test and therefore wouldn't be run any different. Will update PR soon.